test/e2e/edgedevice_test.go

```diff
@@ -212,7 +212,7 @@ func (e *edgeDeviceDocker) waitForDevice(cond func() bool) error {
 
 func (e *edgeDeviceDocker) Register() error {
 	ctx := context.Background()
-	resp, err := e.cli.ContainerCreate(ctx, &container.Config{Image: EdgeDeviceImage}, &container.HostConfig{Privileged: true}, nil, nil, e.name)
+	resp, err := e.cli.ContainerCreate(ctx, &container.Config{Image: EdgeDeviceImage}, &container.HostConfig{Privileged: true, ExtraHosts: []string{"project-flotta.io:172.17.0.1"}}, nil, nil, e.name)
```

Comment:
must the IP address be hard coded? how to retrieve it?

Reply:
it's the IP given by the docker-network, is always the same, the gateway one.